### PR TITLE
Expand coverage of parser tests.

### PR DESCRIPTION
**Pull request by siarheishut**

### CODE REVIEW

File: tests/request_parser_test.cpp [hunk 1]
```diff
@@ -10,6 +10,127 @@
...
+          .input = json::Dict{
+              {"type", "Bus"},
+              {"name", "Bus 1"},
+              {"stops", json::List{"stop1", "stop2", "stop1"}},
+              {"is_roundtrip", true}},
+          .want_fail = false
+      },
+      TestCase{
+          .name = "Valid PostStopRequest",
+          .input = json::Dict{
```
Comment (foxfrikses) [inline tests/request_parser_test.cpp:61]: I would move tests, that test 'name', 'longitude' and other field tests here as well

File: tests/request_parser_test.cpp [hunk 2]
```diff
@@ -10,6 +10,127 @@
...
+          .name = "Valid GetStopRequest",
+          .input = json::Dict{{"type", "Stop"}, {"name", "1"}, {"id", 1}},
+          .want_fail = false
+      },
+      TestCase{
+          .name = "Valid GetRouteRequest",
+          .input = json::Dict{
+              {"type", "Route"},
+              {"from", "1"},
+              {"to", "2"},
```
Comment (foxfrikses) [inline tests/request_parser_test.cpp:117]: Same as in comment above

**Review by foxfrikses — APPROVED** [review]
LGTM

File: tests/request_parser_test.cpp (hunk 1 above)
Comment (siarheishut) [reply tests/request_parser_test.cpp:61]: But these tests check if the common properties are valid, while `name`, `longitude` etc are individual properties. Are you sure I should move it?

**Review by foxfrikses — APPROVED** [review]
LGTM

**Pull request merged**